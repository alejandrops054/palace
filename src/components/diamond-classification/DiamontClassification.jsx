/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react'
import Table from 'rc-table'
import {Card, CardBody, CardTitle, CardHeader,Row, Col, CardFooter} from 'reactstrap'
import {Button as ButtonUI} from '../index'

const DiamontClassification = () => {
    const columns = [
        { title: 'Code', dataIndex: 'code', key: '1', width: 200},
        { title: 'Description', dataIndex: 'description', key: '2', width: 200 },
      ]
      const data = [
          {code:'2201', description: ''},
          {code:'2201', description: ''},
          {code:'2201', description: ''}
      ]
    return (
        <div>
            <Card>
                <CardHeader><CardTitle tag="h3">Diamond Classifications</CardTitle></CardHeader>
                <CardBody>
                    <Row>
                        <Table style={{width:'auto'}} className="table" id="resume-table" columns={columns} data={data}/>
                    </Row>
                </CardBody>
                <CardFooter>
                            <div className="d-flex justify-content-end">
                                <ButtonUI col={2}
                                    icon="fas fa-clipboard-list"
                                    title="Log"
                                    typebootstrap="link"
                                    type="button"
                                  
                                    />
                                <ButtonUI col={2}
                                    icon="fas fa-plus"
                                    title="Add"
                                    typebootstrap="link "
                                    />
                                     <ButtonUI col={2}
                                    icon="fas fa-times"
                                    title="Properties"
                                    typebootstrap="link"
                                    type="button"
                                  
                                    />
                                     <ButtonUI col={2}
                                    icon="fas fa-sync-alt"
                                    title="Refresh"
                                    typebootstrap="link"
                                    type="button"
                                    />
                                <ButtonUI col={2}
                                    icon="fas fa-times"
                                    title="Close"
                                    typebootstrap="link "
                                   />
                            </div>
                        </CardFooter>
            </Card>
        </div>
    )
}

export default DiamontClassification
