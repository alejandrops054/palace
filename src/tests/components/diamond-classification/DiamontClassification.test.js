/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/

import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import DiamontClassification from '../../../components/diamond-classification/DiamontClassification'

describe('DiamontClassification component', () => {
    test('debe mostrar snapshot', () => {

        const wrapper = shallow(<DiamontClassification/>)
        expect(wrapper).toMatchSnapshot();
    })

})