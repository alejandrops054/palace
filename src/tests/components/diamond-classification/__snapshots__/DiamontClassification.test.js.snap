// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DiamontClassification component debe mostrar snapshot 1`] = `
<div>
  <Card
    tag="div"
  >
    <CardHeader
      tag="div"
    >
      <CardTitle
        tag="h3"
      >
        Diamond Classifications
      </CardTitle>
    </CardHeader>
    <CardBody
      tag="div"
    >
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Table
          className="table"
          columns={
            Array [
              Object {
                "dataIndex": "code",
                "key": "1",
                "title": "Code",
                "width": 200,
              },
              Object {
                "dataIndex": "description",
                "key": "2",
                "title": "Description",
                "width": 200,
              },
            ]
          }
          data={
            Array [
              Object {
                "code": "2201",
                "description": "",
              },
              Object {
                "code": "2201",
                "description": "",
              },
              Object {
                "code": "2201",
                "description": "",
              },
            ]
          }
          emptyText={[Function]}
          id="resume-table"
          prefixCls="rc-table"
          rowKey="key"
          style={
            Object {
              "width": "auto",
            }
          }
        />
      </Row>
    </CardBody>
    <CardFooter
      tag="div"
    >
      <div
        className="d-flex justify-content-end"
      >
        <Button
          col={2}
          icon="fas fa-clipboard-list"
          title="Log"
          type="button"
          typebootstrap="link"
        />
        <Button
          col={2}
          icon="fas fa-plus"
          title="Add"
          typebootstrap="link "
        />
        <Button
          col={2}
          icon="fas fa-times"
          title="Properties"
          type="button"
          typebootstrap="link"
        />
        <Button
          col={2}
          icon="fas fa-sync-alt"
          title="Refresh"
          type="button"
          typebootstrap="link"
        />
        <Button
          col={2}
          icon="fas fa-times"
          title="Close"
          typebootstrap="link "
        />
      </div>
    </CardFooter>
  </Card>
</div>
`;
